src/review.rs

```diff
@@ -603,6 +611,26 @@
         assert_eq!(r.status().unwrap(), ReviewStatus::Submitted);
     }
 
+    #[test]
+    fn test_review_validation_with_pr_description() {
+        let review = include_str!("../testdata/review/pr_description/review");
+        let metadata = include_str!("../testdata/review/pr_description/metadata");
+        let (r, _dir) = setup(review, metadata);
+
+        r.validate_review_file(review)
+            .expect("Failed to validate review file with PR description");
+    }
+
+    #[test]
+    fn test_review_validation_with_interleaving_pr_description() {
+        let review = include_str!("../testdata/review/pr_description_interleaving/review");
+        let metadata = include_str!("../testdata/review/pr_description_interleaving/metadata");
+        let (r, _dir) = setup(review, metadata);
+
+        r.validate_review_file(review)
+            .expect("Failed to validate review file with interleaving PR description");
+    }
+
```

Comment:
These tests look good; thanks for adding them. But could you please also add tests in `parser.rs` that check that the appropriate comments were parsed out? It would be good to also validate that the parser can handle PR description.

Reply:
I have added some tests which required changes to the parser because they confirmed that some of the comments were not parsed out.

Please have another look!